cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/newrelic/DefaultNewRelicQuerySupplier.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.monitor.sampling.newrelic;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.linkedin.kafka.cruisecontrol.metricsreporter.metric.RawMetricType;
+
+import static com.linkedin.kafka.cruisecontrol.metricsreporter.metric.RawMetricType.*;
+
+/**
+ * Contains the NRQL queries which will output broker, topic, and partition level
+ * stats which are used by cruise control.
+ */
+public final class DefaultNewRelicQuerySupplier implements NewRelicQuerySupplier {
+    private static final HashMap<String, RawMetricType> BROKER_METRICS = new HashMap<>();
+    private static final HashMap<String, RawMetricType> TOPIC_METRICS = new HashMap<>();
+    private static final HashMap<String, RawMetricType> PARTITION_METRICS = new HashMap<>();
```

Comment:
I think these can be instance variables

Reply:
NVM these should be static - you were correct

Reply:
We should check if there's some sort of ImmutableMap available that we can use here

Reply:
If yes, we can definitely just make these `public static final`